cpp/src/aztec/honk/proof_system/prover.cpp

```diff
@@ -299,6 +300,17 @@ template <typename settings> void Prover<settings>::execute_relation_check_round
                                         sumcheck::GrandProductInitializationRelation>;
 
     // Compute alpha challenge
+    transcript.apply_fiat_shamir("zeta");
```

Comment:
This naming is now conflicting with what we call zeta_challenge  inShplonk. That's probably my fault - when I proposed zeta in that context I was forgetting about this zeta. We should change that back to z_challenge or something.

Reply:
I think we caught that at the time? I'm not seeing another "zeta".